tests/api/worlds/track.wbt

```diff
@@ -1,6 +1,7 @@
 #VRML_SIM R2022a utf8
 WorldInfo {
   basicTimeStep 8
+  coordinateSystem "NUE"
```

Comment:
Do we really want this?

Reply:
Nearly all the tests are still `NUE`. I don't know if it is worth to convert them.